.github/actions/ref_from_sha/test/e2e.cjs

```diff
@@ -1,7 +1,7 @@
 const github = require("@actions/github");
 
-const prFromSha = require("../pr_from_sha");
-const branchFromSha = require("../branch_from_sha");
+const prFromSha = require("../pr_from_sha.cjs");
```

Comment:
Potentially we could refactor this all to use ESM but this is the minimal change to get this to work with the new requirement to use .cjs extensions for commonjs code